cmd/notation/plugin/install.go

```diff
@@ -330,6 +331,23 @@ func installPluginWithOptions(ctx context.Context, opts plugin.CLIInstallOptions
 		if errors.As(err, &errPluginDowngrade) {
 			return fmt.Errorf("%w.\nIt is not recommended to install an older version. To force the installation, use the \"--force\" option", errPluginDowngrade)
 		}
+
+		var errExeFile plugin.PluginExecutableFileError
+		if errors.As(err, &errExeFile) {
+			var errPath *os.PathError
+			if errors.As(errExeFile, &errPath) && errPath.Op == "fork/exec" {
+				return fmt.Errorf("%s.\nPlease ensure the plugin is executable and meet the installation requirements on the %s/%s.",
+					// hide the wrapped error message to simplify the output
+					strings.TrimSuffix(strings.Split(errExeFile.Error(), "fork/exec")[0], ": "),
```

Comment:
consider using `strings.Cut(errExeFile.Error(), ': fork/exec')`

Reply:
Based on offline discussion, the `plugin.PluginExecutableFileError` should be updated to include an `innerError`.